docs/parameters/validation.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Validation
+description: Validation of pipeline parameters at run time.
+---
+
+# Validation of pipeline parameters
```

Comment:
can this be run standalone before a pipeline run and not essentially as part of the main run ?

Reply:
It has to be run within a pipeline run to validate the provided parameters. But doesn't need to be part of the workflow.

Reply:
You could maybe add some kind of flag e.g. `--validate` and write some code yourself that checks if that parameter is set. When it's set you can run the validation and exit the pipeline, but I'm not sure why you would want to do that?

Reply:
well the idea is of a dry-run....im thinking after pipeline releases..validate....then run

Reply:
I'm afraid I'm not really following? Validation will happen before actually running the pipeline?

Reply:
We have `nf-core schema lint` to check that the schema are valid. That's outside the scope of this repo though.